### DISCUSSION TITLE
Using big RootFS image for Xavier devkit (sdcard based)

### DISCUSSION THREAD

**Discussion by tirankas** [OE4T/meta-tegra#714] [Q&A] [answered]
Hi Guys,
Any idea where this error comes from:
| [   3.1484 ] Copying signatures
| [   3.1497 ] tegrahost_v2 --chip 0x19 0 --partitionlayout flash.xml.bin --updatesig images_list_signed.xml
| [   4.5226 ]
| [   4.5248 ] tegraparser_v2 --generategpt --pt flash.xml.bin
| [   4.5270 ] End sector for APP, expected at: 33554398, actual: 57332775
| [   4.5276 ]
| Error: Return value 4
| Command tegraparser_v2 --generategpt --pt flash.xml.bin
| WARNING: exit code 1 from a shell command.
| 
ERROR: Task (/home/tirank/GIT/nvidia_1.0.5_new/tegra-l4t/layers/meta-tegra/recipes-kernel/kernel-bup-payload/kernel-bup-payload.bb:do_deploy) failed with exit code '1'
NOTE: Tasks Summary: Attempted 6753 tasks of which 6736 didn't need to be rerun and 2 failed.

I am trying to build for an sdcard based Xavier, SD size is 64G (TEGRAFLASH_SDCARD_SIZE = "64G").
rootfs size set via ROOTFSPART_SIZE_jetson-xavier-nx-devkit = "7516192768"

Any limitation I should be aware of?

Thanks

**Comment by ichergui** [comment, marked as answer]
Hey @tirankas 

Please refer to this [wiki page](https://github.com/OE4T/tegra-demo-distro/wiki/Customization-of-Storage-Size-and-root-filesystem-size-on-Mender-Builds) for rootfs setting size.

**Comment by tirankas** [comment]
Thanks, works fine now...
